kubectl-apply-all.sh

```diff
@@ -6,5 +6,6 @@
 #   * When the cluster is first created
 #   * Whenever the configuration for any resource has been updated
 
-# Apply the base Soucegraph deployment 
-kubectl apply --prune -l deploy=sourcegraph -f base --recursive
```

Comment:
the removal of prune means we no longer will remove no longer used services?

Reply:
It currently doesn't work anyway; https://github.com/kubernetes/kubernetes/issues/66430

Reply:
i had no luck with apply -k --prune. it did worse and failed to create any objects

Reply:
I think this a rough problem. It doesn't just apply to services, but _any_ resource that's no longer referenced.

Reply:
this was solved in a different PR